[Fix] update lr_updater.py links

doem97 opened the pull request:
replace out-of-date links with master branch links, resolve #351 

## Motivation

Original document's lr_updater.py [link](https://github.com/open-mmlab/mmcv/blob/f48241a65aebfe07db122e9db320c31b685dc674/mmcv/runner/hooks/lr_updater.py#L227) and CosineAnnealingLrUpdaterHook directs to a deprecated version (whose name was still Cosine**An**ealing with only one "n")

## Modification

I update the lr_updater.py link to the [master branch](https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/hooks/lr_updater.py), and the CosineAnnealingLrUpdaterHook() link to [current version's](https://github.com/open-mmlab/mmcv/blob/0092699fef27a0e6cbe9c37f4c4de2fb6996a1c7/mmcv/runner/hooks/lr_updater.py#L269).

## Checklist

**Before PR**:

- [x] Pre-commit or other linting tools are used to fix the potential lint issues.
- [x] Bug fixes are fully covered by unit tests, the case that causes the bug should be added in the unit tests.
- [x] The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [x] The documentation has been modified accordingly, like docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with those projects, like MMDet or MMSeg.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

doem97 committed:
update lr_updater.py file path

replace out-of-date links with master branch link

YuanLiuuuuuu commented on docs/en/tutorials/4_schedule.md:
````diff
@@ -48,9 +48,9 @@ lr_config = dict(
 ```
 
-Then during training, the program will call [CosineAnealingLrUpdaterHook](https://github.com/open-mmlab/mmcv/blob/f48241a65aebfe07db122e9db320c31b685dc674/mmcv/runner/hooks/lr_updater.py#L227) periodically to update the learning rate.
+Then during training, the program will call [CosineAnealingLrUpdaterHook](https://github.com/open-mmlab/mmcv/blob/0092699fef27a0e6cbe9c37f4c4de2fb6996a1c7/mmcv/runner/hooks/lr_updater.py#L269) periodically to update the learning rate.
````
Better change the link to the [stable version](https://github.com/open-mmlab/mmcv/blob/0092699fef27a0e6cbe9c37f4c4de2fb6996a1c7/mmcv/runner/hooks/lr_updater.py#L256)

doem97 replied:
Thanks, your provided link is the same as I provided one (blob/0092....)

doem97 replied:
Reference to code/line must be a blob link, while ref to file should be stable link.

For example, 
1. ref to line: `stable_link/lr_updater.py#L269` is dangerous as the file is changing, L269 may ref to the wrong code. It must be `blob/xxx/lr_updater.py#L269`
2. ref to file: `stable_link/lr_updater.py` is the file. When file deleted the link directs to 404, which won't cause misunderstanding.

fangyixiao18 commented on docs/en/tutorials/4_schedule.md:
````diff
@@ -48,9 +48,9 @@ lr_config = dict(
 ```
 
-Then during training, the program will call [CosineAnealingLrUpdaterHook](https://github.com/open-mmlab/mmcv/blob/f48241a65aebfe07db122e9db320c31b685dc674/mmcv/runner/hooks/lr_updater.py#L227) periodically to update the learning rate.
+Then during training, the program will call [CosineAnealingLrUpdaterHook](https://github.com/open-mmlab/mmcv/blob/0092699fef27a0e6cbe9c37f4c4de2fb6996a1c7/mmcv/runner/hooks/lr_updater.py#L269) periodically to update the learning rate.
````
[CosineAnealingLrUpdaterHook] -> [CosineAnnealingLrUpdaterHook]
I think here we also need to fix the typo

doem97 replied:
Please review, I made related changes. Thanks.

doem97 committed:
Update code reference to newest version

1. fix typo err of `CosineAnnealingLrUpdaterHook`
2. update code ref links to the newest version (including `CosineAnnealingLrUpdaterHook`, `CyclicLrUpdaterHook`, `CyclicMomentumUpdaterHook`).

YuanLiuuuuuu commented:
The master branch has been updated. Please rebase your branch to the latest commit of master. Thanks!

doem97 commented:
> The master branch has been updated. Please rebase your branch to the latest commit of master. Thanks!

Thanks, is it ok now?

YuanLiuuuuuu commented:
Please rebase your branch onto master, instead of merging. To fix it, please follow these steps below:
- git reset 963096d
- git branch -D -r origin/patch-1
- git rebase master
- git push origin patch-1

doem97 commented:
> Please rebase your branch onto master, instead of merging. To fix it, please follow these steps below:
> 
> * git reset [963096d](https://github.com/open-mmlab/mmselfsup/commit/963096da2afa46bc8475578595dbb591215f135b)
> * git branch -D -r origin/patch-1
> * git rebase master
> * git push origin patch-1

Thanks. I rebase and pushed to origin/patch-1. Pls check, thanks!

YuanLiuuuuuu commented:
Thanks! LGTM.
